packages/terra-site/src/examples/button/ButtonTheme.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ThemeProvider from 'terra-theme-provider';
+import Button from 'terra-button';
+
+class ButtonTheme extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      theme: '',
+      isAnimated: false,
+    };
+    this.handleThemeChange = this.handleThemeChange.bind(this);
+    this.handleRandomThemeChange = this.handleRandomThemeChange.bind(this);
+    this.handleAutoThemeChange = this.handleAutoThemeChange.bind(this);
+  }
+
+  handleThemeChange(e) {
+    this.setState({ theme: e.target.value });
+  }
+
+  handleRandomThemeChange() {
+    const concept = Math.ceil(Math.random() * 6);
+    this.setState({ theme: `terra-consumer-theme-concept${concept}` });
+  }
+
+  handleAutoThemeChange() {
+    if (this.state.isAnimated) {
+      clearInterval(this.state.animateID);
+      this.setState({ isAnimated: false });
+    } else {
+      this.setState({
+        animateID: setInterval(this.handleRandomThemeChange, 500),
+        isAnimated: true,
+      });
+    }
+  }
+
+  render() {
+    let themeSwitcher;
+
+    function supportsCSSVars() {
+      return window.CSS && window.CSS.supports && window.CSS.supports('--fake-var', 0);
+    }
+
+    if (supportsCSSVars()) {
+      themeSwitcher = (
+        <div style={{ marginBottom: '1rem' }}>
+          <label htmlFor="theme"> Theme: </label>
+          <select value={this.state.theme} onChange={this.handleThemeChange}>
+            <option value="">Default</option>
+            <option value="terra-consumer-theme-concept1">terra-consumer-theme-concept1</option>
+            <option value="terra-consumer-theme-concept2">terra-consumer-theme-concept2</option>
+            <option value="terra-consumer-theme-concept3">terra-consumer-theme-concept3</option>
+            <option value="terra-consumer-theme-concept4">terra-consumer-theme-concept4</option>
+            <option value="terra-consumer-theme-concept5">terra-consumer-theme-concept5</option>
+            <option value="terra-consumer-theme-concept6">terra-consumer-theme-concept6</option>
+            <option value="terra-consumer-theme-neutral">terra-consumer-theme-neutral</option>
+          </select>
+        </div>
+      );
+    } else {
+      themeSwitcher = (
+        <div />
+      );
+    }
+
+    const rainbowButton = this.state.isAnimated ? 'Click me to stop' : "Don't click me!";
```

Comment:
Can we change this to `Click to animate all themes` or something like that? @mmalaker What are your thoughts?